### PR TITLE
tfm_(psa|regression)_test: Add tags to each test

### PR DESCRIPTION
**Pull request by Mangl10**

### CODE REVIEW

File: tests/tfm/tfm_psa_test/testcase.yaml [hunk 1]
```diff
@@ -13,28 +13,38 @@ common:
       - "TOTAL FAILED *: 0"
 tests:
   tfm.psa_test_protected_storage_lvl1:
+    tag: lvl1
```
Comment (torsteingrindvik) [inline tests/tfm/tfm_psa_test/testcase.yaml:16]: I think it's `tags` not `tag`

Comment (Mangl10) [reply tests/tfm/tfm_psa_test/testcase.yaml:16]: Yes, missed that.

**Review by torsteingrindvik — APPROVED** [review]

**Review by joerchan — CHANGES_REQUESTED** [review]
If this is a generic tag in global scope taking lvl1/2 for this seems not very future proof. I suggges tfm_lvl1/2

**Comment by gmarull** [comment]
please user forks, branches in upstream repo are not allowed.

**Pull request closed without merging**